infrastructure/template.yaml

```diff
@@ -632,6 +632,14 @@ Resources:
       KmsMasterKeyId: !Ref SnsKmsKey
       TopicName: UserAccountDeletion
 
+ UserAccountDeletionTopicArnParameter:
```

Comment:
The linter's not happy - I think it's your indentation here

Reply:
yeah just spotted that too, thanks, switching to the machine where I have `cfn-lint` installed 😭